index.html

```diff
@@ -0,0 +1,41 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <div class="header">
+        <div>
+            <nav>
+                <div class="logo"><a href="">ECHEZONA</a></div>
+                <a href="#" class="toggle-button">
+                    <svg class="bar" width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z" fill="#6070FF"/>
+                    </svg>
+                </a>
+            </nav>
+            
+            <div class="head-body-container">
+                <div class="head-body">
+                    <h1>I’m Echezona <br> Glad to see you!</h1>
+                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <h3>LET'S CONNECT</h3>
+                    <ul>
+                        <li><a href=""><img src="image/Vector.png" alt=""></a></li>
+                        <li><a href=""><img src="image/Vector(1).png" alt=""><img src="image/Vector(2).png" alt=""></a></li>
+                        <li><a href=""><img src="image/Vector(3).png" alt=""></a></li>
+                        <li><a href=""><img src="image/Vector(4).png" alt=""></a></li>
+                        <li><a href=""><img src="image/Vector(5).png" alt=""></a></li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </div>
+
+
+</body>
```

Comment:
- [ ] You have made a great effort to fix this project for approval. well done so far. There is still a lot that needs to fix. First of all, your HTML Structure is wrong and causing you the issue you have on the project. See the structure below for a guide. Please use this HTML structure and apply flex to `header` . Also, apply flex to `section` tag  but use `flex-direction: column;`

```
<header>
    <a href="index.html">Logo</a>
    <nav>
      <img src="hamburger menu icon" alt="hamburger">
    </nav>
  </header>
  <section>
    <h1>I'm Mariane, Glad to see you</h1>
    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Laboriosam, itaque?</p>
    <a href="link">LET'S CONNECT</a>
    <ul>
      <li>
        <a href="">Twitter logo</a>
      </li>
      <li>
        <a href="">Other logo's</a>
      </li>
      <li>
        <a href="">Other logo's</a>
      </li>
      <li>
        <a href="">Other logo's</a>
      </li>
      <li>
        <a href="">Other logo's</a>
      </li>
    </ul>
  </section>
```

Your background illustration will be attached with CSS using the `background-image` property. Please try to make the project look as much as the Figma design.